browser/gui/app.cpp

```diff
@@ -156,6 +157,35 @@ void App::set_scale(unsigned scale) {
     engine_.set_layout_width(windowSize.x / scale_);
 }
 
+void App::sticky_windows(sf::Event::SizeEvent size) {
+    const char *names[] = {
+        "Navigation",
+        "HTTP Response",
+        "DOM",
+        "Stylesheet",
+        "Layout",
+        NULL,
+    };
+    
+    int delta = k_width - size.width;
```

Comment:
This is UB on the first call to this function since k_width isn't initialized.

Reply:
Aren't integer members 0 if NOS in c++?

Reply:
No, unless set in e.g. a constructor they're uninitialized and reading from them is UB.

Reply:
good enough?